src/hooks/useValidators.ts

```diff
@@ -40,11 +50,27 @@ const useValidators = () => {
     return true
   }
 
+  const tagInputValidator: Validate<
+    string | ObjectType | undefined,
+    FormValues
+  > = (value) => {
+    const inputValue = typeof value === 'string' ? value : ''
+
+    if (!inputValue || hasValueOver50Chars(inputValue))
```

Comment:
**[non-blocker]**

Not a big issue, but the first "if" statement here already catches the case, where inputValue is falsey, which means that the input length error will be displayed in that case. There is no need to check it in the other "if" cases